src/components/HomePage/HomePageImage.tsx

```diff
@@ -1,7 +1,7 @@
 import Image from 'next/image';
 
 import styles from './HomePageImage.module.scss';
-import PrayerTimes from './PrayerTimes';
+import PrayerTimes from './PrayerTimes/PrayerTimes';
```

Comment:
1. Will do it in another PR, need to research a bit.
2. Tried to make it consistent with the other button, doesn't looks good.
3. ✅
4. removed toast from settings, since we are already using radio now, so there's already a feedback for user (vs previous implementation using button)
5. ✅